SYCL/GroupAlgorithm/SYCL2020/sort.cpp

```diff
@@ -2,18 +2,16 @@
 // RUN: %CPU_RUN_PLACEHOLDER %t.out
 // RUN: %GPU_RUN_PLACEHOLDER %t.out
 // RUN: %ACC_RUN_PLACEHOLDER %t.out
-//
-// RUNx: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -I . -o %t13.out
 
 #include "support.h"
-#include <CL/sycl.hpp>
+#include <sycl/sycl.hpp>
 
 #include <algorithm>
 #include <iostream>
 #include <random>
 #include <vector>
 
-namespace my_sycl = sycl::ext::oneapi::experimental;
+using namespace sycl::ext::oneapi::experimental;
```

Comment:
Why the namespace alias was replaced by `using namespace`? It's saved just some symbols, but makes the code clear

Reply:
Just because I consider naming "my_*" variables to be bad style. :)
Do you really think that using `my_sycl::func` instead of `func` makes code clear?

Reply:
From my perspective having some namespace (not `my_sycl`, it can be renamed) is better than no namespace before `func`. However, it's a minor comment that doesn't prevent merging. Personally, I just don't like any `using namespace` in the code since I faced some issues previously.

Reply:
Replaced `my_sycl` with `oneapi_exp`.